packages/camera/camera/test/camera_image_stream_test.dart

```diff
@@ -20,12 +20,19 @@ void main() {
   });
 
   test('startImageStream() throws $CameraException when uninitialized', () {
-    final CameraController cameraController = CameraController(
-        const CameraDescription(
-            name: 'cam',
-            lensDirection: CameraLensDirection.back,
-            sensorOrientation: 90),
-        ResolutionPreset.max);
+    final CameraController cameraController = CameraController.withSettings(
+      const CameraDescription(
+          name: 'cam',
+          lensDirection: CameraLensDirection.back,
+          sensorOrientation: 90),
+      mediaSettings: const MediaSettings(
+        resolutionPreset: ResolutionPreset.low,
+        fps: 15,
+        videoBitrate: 200000,
+        audioBitrate: 32000,
+        enableAudio: true,
```

Comment:
Why does every test need to fully specify every value? If we can't make a camera controller without specifying a lot of values we don't care about, then we have a significant API problem.

Reply:
not necessary, removed